src/libraries/System.Private.CoreLib/src/Internal/Console.Android.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Internal
+{
+    public static partial class Console
+    {
+        public static unsafe void Write(string s)
+        {
+            Interop.Logcat.AndroidLogPrint(Interop.Logcat.LogLevel.Debug, "DOTNET", s);
```

Comment:
Is Internal.Console.Write something that we anticipate being used to debug CoreLib in full XA applications, or just in the runtime?  If the latter, we should ping the Android team - I think the default logcat filter in VS (which I think comes from XA) is something like `Xamarin:*`, `Mono:*`, and `(appname):*`, I think.  We could ask to also add `DOTNET`

Reply:
I believe this is just exclusive to runtime as of right now.  That said, we should add DOTNET.  Our crypto interop has been using that label for a while.

Reply:
It is just for internal use by the .NET runtime teams. It is not expected to be used by Android team or customers - they should us standard System.Console.